### PR TITLE
This commit includes implementation and comparison of Bernoulli and Gaussian Dropout on Fully Connected NN.

### PR DESCRIPTION
**Pull request by Marselka**

### CODE REVIEW

**Comment by maxblumental** [comment]
Could you please remove `data` folder.

- [x] remove MNIST data

**Comment by maxblumental** [comment]
You could make the code a little bit more readable:
- remove the commented code;
- remove unnecessary information from the output (e.g. `train loss: 0.000000`).

**Comment by maxblumental** [comment]
How about to hide the training logging (`==>>> epoch: 0, batch index: 100, train loss: 0.000000`)? It's not very informative when you look at the code in the repository.

**Comment by maxblumental** [comment]
FYI: [sklearn.metrics.accuracy_score](http://scikit-learn.org/stable/modules/generated/sklearn.metrics.accuracy_score.html)
